uint16[] id_list
float32[] input_torque_list
